specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/keys.json

```diff
@@ -1033,80 +1091,17 @@
     }
   },
   "definitions": {
-    "KeyReleaseCondition": {
-      "properties": {
-        "claim": {
-          "x-ms-client-name": "claimType",
-          "type": "string",
-          "minLength": 1,
-          "description": "claim type name"
-        },
-        "condition": {
-          "x-ms-client-name": "claimCondition",
-          "type": "string",
-          "minLength": 1,
-          "description": "condition to test",
-          "enum": [
-            "equals"
-          ],
-          "x-ms-enum": {
-            "name": "KeyReleaseConditionCondition",
-            "modelAsString": true,
-            "values": [
-              {
-                "value": "equals",
-                "description": "equals comparison."
-              }
-            ]
-          }
-        },
-        "value": {
-          "type": "string"
-        }
-      }
-    },
-    "KeyReleaseAuthority": {
-      "properties": {
-        "authority": {
-          "x-ms-client-name": "authorityURL",
-          "type": "string",
-          "minLength": 1,
-          "description": "Base URL of the attestation service."
-        },
-        "allOf": {
-          "type": "array",
-          "items": {
-            "$ref": "#/definitions/KeyReleaseCondition",
-            "description": "The set of claim conditions that must be met."
-          }
-        }
-      }
-    },
     "KeyReleasePolicy": {
       "properties": {
-        "version": {
-          "description": "key release policy version",
+        "contentType": {
+          "description": "Content type and version of key release policy",
```

Comment:
Should reword this to fit the value, like "Content type, policy schema version, and optional content encoding of the key release policy"

Reply:
This was just copied over from the original 7.2-preview file.

Reply:
Understood, but we've collectively made doc improvements in the past. ;) Consider this a nit, though.